nifi-assembly/pom.xml

```diff
@@ -470,6 +470,15 @@ language governing permissions and limitations under the License. -->
     </properties>
     <profiles>
         <profile>
+            <id>mapr</id>
```

Comment:
Seems like with the addition of the custom_hadoop_libraries profile and the nifi.hadoop.distro.params, we don't need this explicit profile anymore (or it should be renamed to match the one in the NAR)? I like the generic profile/options versus anything vendor-specific